packages/vx-demo/src/components/Footer.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export default () => (
   <div className="footer">
     <div>
-      <img src="static/favicon.ico" alt="vx" />
+      <img src="/static/favicon.ico" alt="vx" />
```

Comment:
all of these 'static => `/static` changes are needed for correct URLs within the nested `next` pages like `pages/docs/Axis`